tools/run-clang-tidy-in-ci.sh

```diff
@@ -38,11 +38,12 @@ fi
 # Run Clang-Tidy
 # The negative filters below are to exclude files that include onnx_pb.h or
 # caffe2_pb.h, otherwise we'd have to build protos as part of this CI job.
-time python tools/clang_tidy.py                  \
-  --verbose                                      \
-  --paths torch/csrc/                            \
-  --diff "$BASE_BRANCH"                          \
-  -g"-torch/csrc/jit/export.cpp"                 \
-  -g"-torch/csrc/jit/import.cpp"                 \
-  -g"-torch/csrc/jit/netdef_converter.cpp"       \
+time python tools/clang_tidy.py                     \
+  --verbose                                         \
+  --paths torch/csrc/                               \
+  --diff "$BASE_BRANCH"                             \
+  -g"-torch/csrc/jit/export.cpp"                    \
+  -g"-torch/csrc/jit/import.cpp"                    \
+  -g"-torch/csrc/jit/netdef_converter.cpp"          \
+  -g"-torch/csrc/jit/passes/onnx/cast_constant.cpp" \
```

Comment:
I prefer to let clang tidy cover as many files as possible. So let's try to remove the dependence of onnx header files in cast_constant.cpp file. Since there are only two places, hard coding with comments sounds acceptable.